### PR TITLE
[edn/doc] remove references to broadcast mode

### PR DESCRIPTION
**Pull request by mwbranstad**
As part of the review process, broadcast mode was dropped.

The broadcast timer shows up in the graphic.

The request/ack waveform was updated.

Signed-off-by: Mark Branstad <mark.branstad@wdc.com>

### CODE REVIEW

File: hw/ip/edn/doc/_index.md [hunk 1]
```diff
@@ -203,13 +203,12 @@ The following waveform shows an example of how the peripheral hardware interface
 {signal: [
    {name: 'clk'           , wave: 'p...|.........|.......'},
    {name: 'edn_enable'    , wave: '01..|.........|.......'},
-   {name: 'edn_req'       , wave: '0..1|......0.1|.....0.'},
-   {name: 'edn_ack'       , wave: '0...|.1.01.0..|101....'},
-   {name: 'edn_bus[31:0]' , wave: 'x...|.33.33...|3.3333.', data: ['gb0','gb1','gb2','gb3','es4','es5','es6','es7','es8']},
+   {name: 'edn_req'       , wave: '0..1|..01.0..1|.....0.'},
+   {name: 'edn_ack'       , wave: '0...|.10.10...|....10.'},
+   {name: 'edn_bus[31:0]' , wave: 'x...|.3x.3x...|....3x.', data: ['es0','es1','es2']},
```
Comment (martin-lueker) [inline hw/ip/edn/doc/_index.md:208]: These should not be "x"'s.  You may not care what the value is, but some IP consuming them might.

File: hw/ip/edn/doc/edn_top_diag.svg [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <!-- Generated by Microsoft Visio, SVG Export edn_top_diag.svg Page-1 -->
```
Comment (martin-lueker) [inline hw/ip/edn/doc/edn_top_diag.svg:3]: Wait, I'm confused.  This looks like you are putting the broadcast back into this figure.  What is your intent here?

**Review by martin-lueker — CHANGES_REQUESTED** [review]
See other comments.  
Also, did you view it on a Linux browser by chance?

**Review by tjaychen — APPROVED** [review]

File: hw/ip/edn/doc/edn_top_diag.svg (hunk 2 above)
Comment (mwbranstad) [reply hw/ip/edn/doc/edn_top_diag.svg:3]: Yes, this is in error. Need to update this diagram first.

File: hw/ip/edn/doc/_index.md (hunk 1 above)
Comment (mwbranstad) [reply hw/ip/edn/doc/_index.md]: Assuming exact details matter for security integrity review, the waveform for request/ack has been updated with implementation specifics - x's have been removed.

File: hw/ip/edn/doc/edn_top_diag.svg (hunk 2 above)
Comment (mwbranstad) [reply hw/ip/edn/doc/edn_top_diag.svg]: During the hand-over process of this PR from @martin-lueker to me, the source (visio) was intack, but the svg file was not re-generated. This has been corrected.

**Review by rswarbrick — APPROVED** [review]

**Pull request merged**